src/engines/kotor/object.h

```diff
@@ -115,6 +128,10 @@ class Object : public Aurora::NWScript::Object, public KotOR::ScriptContainer {
 	bool _static; ///< Is the object static?
 	bool _usable; ///< Is the object usable?
 
+	int _currentHitPoints; ///< The current hitpoints of the object.
+	int _maxHitPoints; ///< The maximum hitpoints of the object.
+	bool _minOneHitPoint; ///< If the object should have at least one hitpoint.
```

Comment:
Initialize those in the constructor, please. To 0, 0, false, I guess. Especially important for _minOneHitPoint, which isn't set in Placeable::loadObject().